cuweb/rds
# WSDEV2546 dependency updates

## What type of PR is this? (check all applicable)

- [X ] Refactor
- [X ] Optimization / Build Tools

## Related Tickets & Documents



- Jira Ticket #WSDEV2546
- Github Issue #

## Is this a breaking change?

- [X ] No

_If yes please note what project will be impacted and describe what is expected to break._

## Have you made an entry in the Changelog

- [X ] Yes

_If no please add as many items as necessary under the "Unreleased" section of the changelog._

## Have you made changes to dependencies?

- [ X] Updated

_If you've made changes to dependencies please note them here, adding if they've been added, removed or updated, including packages version where applicable._

Packages are updated

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, and if applicable add a note
about any UI changes or if a test should be done on a specific browser. Add any other detail relevant to this PR._

Check icons load

## UI accessibility concerns? NO

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. If there are no accessibility concerns delete this section._

## [optional] Are there any post deployment tasks we need to perform?
